CHANGELOG.md

```diff
@@ -1,4 +1,96 @@
+  - New resource **aviatrix_cloudn_transit_gateway_attachment**
+7. Implemented support for setting approved CIDRs in **aviatrix_transit_external_device_conn**:
+  - New attribute ``approved_cidrs``
+8. Implemented support for remote syslog profile name:
+  - New attribute ``name`` in **aviatrix_remote_syslog**
+9. Implemented new resources to decouple ``security_domains`` out of **aviatrix_aws_tgw**:
+  - **aviatrix_aws_tgw_security_domain**
+  - **aviatrix_aws_tgw_security_domain_connection**
+10. Implemented support for intra domain inspection:
+  - New resource **aviatrix_aws_tgw_intra_domain_inspection**
+11. Implemented support for multi-tier transit feature:
+  - New attribute ``enable_multi_tier_transit`` in **aviatrix_transit_gateway** and data source
+12. Implemented support for Alibaba Cloud in resources **aviatrix_account**, **aviatrix_vpc**, **aviatrix_gateway**, **aviatrix_transit_gateway** and **aviatrix_spoke_gateway** and data sources
+13. Implemented support for disabling sending exception emails to Aviatrix:
+  - New resource **aviatrix_controller_email_exception_notification_config**
+14. Implemented support for updating certificate domain:
+  - New resource **aviatrix_controller_cert_domain_config**
+15. Implemented support for TGW segmentation for Egress:
+  - New attribute ``tgw_segmentation_for_egress_enabled`` in **aviatrix_firenet** and data source
+16. Implemented support for OCI FireNet
+17. Implemented support for Egress FireNet route injection:
+  - New attribute ``egress_static_cidrs`` in **aviatrix_firenet** and data source
+18. Implemented support for launching user provided AMI for FireNet:
+  - New attribute ``firewall_image_id`` in **aviatrix_firewall_instance**
+19. Implemented support for optional tunnel ip addresses with following attributes in **aviatrix_site2cloud**:
+  - ``local_tunnel_ip``
+  - ``remote_tunnel_ip``
+  - ``backup_local_tunnel_ip``
+  - ``backup_remote_tunnel_ip``
+20. Implemented support for phase 1 remote identifier for Site2Cloud:
+  - New attribute ``phase1_remote_identifier`` in **aviatrix_site2cloud**
+21. Implemented support for AzureGov, AWSChina and AzureChina clouds in resources **aviatrix_account**, **aviatrix_vpc**, **aviatrix_gateway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+22. Implemented support for creating an **aviatrix_vpc** in Azure with an existing resource group ``resource_group``
+23. Implemented support for separate IAM role and policy for gateways in AWS **aviatrix_account** with new attributes:
+  - ``aws_gateway_role_app``
+  - ``aws_gateway_role_ec2``
+24. Implemented support for IPSec tunnel down detection time in **aviatrix_gatewway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**:
+  - New attribute ``tunnel_detection_time``
+25. Implemented support for **aviatrix_site2cloud** connections to have a single public IP with two gateways:
```

Comment:
25. Implemented single public IP failover support for **aviatrix_site2cloud** connections: